.github/workflows/e2e-tests.yml

```diff
@@ -3,11 +3,16 @@ on: [ deployment_status ]
 jobs:
   prepare-preview:
     # Only trigger for correct environment and status
-    if: ${{ github.event.deployment_status.state == 'success' && (contains(github.event.deployment.environment, '- lightdash PR ') || contains(github.event.deployment.environment, '- lightdash-')) }}
     runs-on: ubuntu-latest
     outputs:
       url: ${{ steps.deployment-url.outputs.url }}
     steps:
+      - name: is render ready
+        # This works assuming "Lightdash PR" is always the last deployment_status to be "success"
+        if: ${{ github.event.deployment_status.state != 'success' || (!contains(github.event.deployment.environment, '- lightdash PR ') || !contains(github.event.deployment.environment, '- lightdash-'))}}
```

Comment:
there is a problem with this logic, has the action is failing with this message:
```Render environment chore-is-render-available-check - lightdash PR #6168 is not ready: success```

the environment and state seem correct

Reply:
Yep, I think the last OR (||) was not correctly wrapped